packages/order-utils/src/order_validation_utils.ts

```diff
@@ -171,6 +179,81 @@ export class OrderValidationUtils {
         const provider = providerUtils.standardizeOrThrow(supportedProvider);
         this._provider = provider;
     }
+
+    // TODO(xianny): remove this method once the smart contracts have been refactored
+    // to return helpful revert reasons instead of ORDER_UNFILLABLE. Instruct devs
+    // to make "calls" to validate order fillability + getOrderInfo for fillable amount.
+    // This method recreates functionality from ExchangeWrapper (@0x/contract-wrappers < 11.0.0)
+    // to make migrating easier in the interim.
+    /**
+     * Validate if the supplied order is fillable, and throw if it isn't
+     * @param erc20ProxyAddress A valid address where an ERC20ProxyContract is deployed
+     * @param erc721ProxyAddress A valid address where an ERC721ProxyContract is deployed
+     * @param exchangeAddress A valid address where the ExchangeContract is deployed
+     * @param zrxTokenAddress A valid address where ZRXTokenContract is deployed
+     * @param networkId The network for all contracts
+     * @param provider The same provider used to interact with contracts
+     * @param signedOrder SignedOrder of interest
+     * @param opts ValidateOrderFillableOpts options (e.g expectedFillTakerTokenAmount.
+     * If it isn't supplied, we check if the order is fillable for the remaining amount.
+     * To check if the order is fillable for a non-zero amount, set `validateRemainingOrderAmountIsFillable` to false.)
+     */
+    public async simpleValidateOrderFillableOrThrowAsync(
+        erc20ProxyAddress: string,
```

Comment:
We should be able to detect these either via an Exchange lookup or from the Exchange address -> contractAddresses -> erc20ProxyAddress.  The exchangeAddress is also on the `signedOrder` so is redundant.

Thoughts on reducing this to just `provider, signedOrder, opts` and looking up internally?